tokens/source/components/status.json

```diff
@@ -0,0 +1,299 @@
+{
+  "status": {
+    "color": {
+      "1na": {
+        "comment": "Color for an unknown 100-199 range response (blue.10).",
+        "value": "{color.alias.blue.10.value}"
+      },
+      "2na": {
+        "comment": "Color for an unknown 200-299 range response (green.10).",
```

Comment:
nit: change "response" to "response status code" in all the comments where appropriate

Reply:
Updated to use "response status code" across the board